vault/resource_approle_auth_backend_role_secret_id.go

```diff
@@ -330,8 +345,8 @@ func approleAuthBackendRoleSecretIDExists(d *schema.ResourceData, meta interface
 	return resp != nil, nil
 }
 
-func approleAuthBackendRoleSecretIDID(backend, role, accessor string, wrapped bool) string {
-	if wrapped {
+func approleAuthBackendRoleSecretIDID(backend, role, accessor string, wrapped bool, reuse_wrapping_token bool) string {
+	if wrapped && !reuse_wrapping_token {
```

Comment:
I wonder if it might make sense to include a special path prefix in the case that `reuseWrappingToken` is true?

Reply:
I'm not sure what you are intending here.. Could you please elaborate on what you mean, if this is required ?

Reply:
I think it should be okay as-is.